### DISCUSSION TITLE
Ways to avoid out of memory error in browser.

### DISCUSSION THREAD

**Discussion by ghost** [victornpb/undiscord#465] [1. Questions & Answers]
I am doing wipe archive and after some time I always receive the out of memory error and the page crashes, I tried multiple browsers, is there some setting I can change to fix this or does the script need to be modified to allow for larger archives?

**Comment by victornpb** [comment]
I have to investigate it, I never experienced this.
You having this problem with what amount of messages roughly? 1k, 2k, 10k, more?
Also what browser are you using, and how much ram are we talking about here?
